pkg/bootstrap/docker/run/run.go

```diff
@@ -215,6 +227,13 @@ func (h *Runner) startContainer(id string) error {
 }
 
 func (h *Runner) runWithOutput() (string, string, int, error) {
+	userNsMode, err := h.dockerHelper.UserNamespaceEnabled()
+	if err != nil {
+		return "", "", 0, err
+	} else if userNsMode {
+		h.hostConfig.Privileged = false
```

Comment:
Why set this false? If the client has called Privileged on this Runner, then UserNsMode would be set to "host", which would work with Privileged set to true, right?

Actually I would get rid of the HostUserns method introduced above and instead, check here if Privileged is set, then if it is, then set h.hostConfig.UsernsMode to "host". Otherwise, leave it alone.

I would think you don't want to set hostConfig.UsernsMode to host if user namespaces is not abled.